frontend/talentsearch/src/js/components/search/deprecated/SearchContainer.tsx

```diff
@@ -200,8 +200,8 @@ export const SearchContainer: React.FC<SearchContainerProps> = ({
             {intl.formatMessage(
```

Comment:
Is there a reason not to reuse the same object from the non-deprecated SearchContainer component? Then you can get rid of `7Eg+38` in fr.json, I think.

```
            {intl.formatMessage(
              {
                defaultMessage: `{candidateCount, plural,
                  one {Results: <testId>{candidateCount}</testId> matching candidate}
                  other {Results: <testId>{candidateCount}</testId> matching candidates}
                }`,
                id: "Fjikio",
                description:
                  "Heading for total matching candidates in results section of search page.",
              },
              {
                testId,
                candidateCount,
              },
            )}
```

Reply:
didn't understand what you meant at first
yeah, sure, deprecated isn't intended to be kept around that long anyway
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4278/commits/eabbe0701f9c3ac09c6351b85dfdb738fce324dd